### ISSUE TITLE
Menu entry for Code generator doesn't appear in Android studio 1.2.2

### ISSUE THREAD

**Issue by d69soft** [tmorcinek/android-codegenerator-plugin-intellij#17]
No menu entry for code generator when right click on any layout file.

**Comment by tmorcinek** [comment]
Thanks for the comment, I will try to use it as soon as I have time. 
Could you please tell me if you are Mac, or Windows user.

**Comment by d69soft** [comment]
Either Mac or Windows user. I tried it on a linux machine with xubuntu.

2015-07-21 14:48 GMT+03:00, Tomasz Morcinek <notifications@github.com>:

**Comment by tmorcinek** [comment]
Does the problem still occur in Android Studio 1.3?

**Comment by d69soft** [comment]
Yes, still no menu entry for code generator appear when right clic on
layout file.

2015-08-05 18:09 GMT+03:00, Tomasz Morcinek <notifications@github.com>:

**Comment by flyou** [comment]
1.5 does not appear yet

**Comment by AlexAndriichenko** [comment]
Works fine in Android Studio 1.4 and 2.0
Maybe you not restart Android Studio after plugin installation?
Menu available after restarting and finishing gradle build

**Comment by erhanbicer** [comment]
Android Studio 2.3 still does not appear

**Comment by YuriHeupa** [comment]
Android Studio 3.0.1 doesn't show option yet.
